draco/tests/test_soft.py

```diff
@@ -5385,11 +5395,13 @@ def test_value_continuous_x():
         list_preferences(
             b.program
             + """
+    attribute((field,interesting),precipitation,true).
```

Comment:
This seems wrong to me. I don't think we want to have people have to add interestedness to all fields just to make the recommendations work.

Reply:
As I interpreted from the kim2018 dataset and their paper, the interestingness is associated with the tasks. something like "identifying the values for some field". So I think if people don't specify the task or interestingness, the recommendation would still work, but probably generate less precise results.

Reply:
I see. Yes, I think it makes sense to say tasks need associated interesting fields.

Reply:
In addition there are soft constraints about only task and mark (without interestingness).